tests/cpp/test_multidevice_pipeline.cpp

```diff
@@ -92,6 +96,9 @@ void PipelineTest::validate(bool validate_with_prescribed_values) {
     auto ref_output =
         shardTensor(ref_unsharded_outputs[i].as<at::Tensor>(), output_tv);
     auto obtained_output = outputs[i].as<at::Tensor>();
+    
+    EXPECT_EQ(ref_output.strides(), obtained_output.strides()) << "Strides are not equal: Ref: " << ref_output.strides() << " Output: " << obtained_output.strides() << std::endl;
```

Comment:
torch::allclose checks for sizes but not strides, so leaving this verification here for future.